File: Lectures/Week 06/Lecture06/src/Car.py
class Car:
    def __init__(self, make = "Lexus", model = "LFA", year = 2010):
        self.make = make
        self.model = model
        self.year = year
    
    def get_make(self):
        return self.make
    
    def set_make(self, make):
        self.make = make
    
    def start(self):
        print(f"Started the {self.make} {self.model}")

    def stop(self):
        print("Car stopped")
    
    def drive(self, speed):
        print(f"Car driving at {speed}")

File: Lectures/Week 06/Lecture06/src/Car.java
public class Car {
    //Fields
    private String make = "";
    private String model = "";
    private int year = 0;

    //Default constructor
    public Car() {
        this.make = "Lexus";
        this.model = "LFA";
        this.year = 2010;
    }

    //Overloaded constructor (When the constructor takes parameters)
    public Car(String make, String model, int year) {
        this.make = make;
        this.model = model;
        this.year = year;
    }

    public String getMake(){
        return this.make;
    }

    public void setMake(String make) {
        this.make = make;
    }

    public void start() {
        System.out.println("Started the " + this.make + " " + this.model);
    }

    public void stop() {
        System.out.println("Car stopped");
    }

    public void drive(int speed) {
        System.out.println("Car driving at " + speed);
    }
}

